client/viewer/js/main.js

```diff
@@ -1,75 +1,95 @@
 /* eslint-env browser */
-/* global console, io*/
-const viewer = require('./viewer.js');
-const port = location.port ? ':'+location.port : '';
-const socket = io('//'+location.hostname+port+'/screens');
-
-console.log('Initialising socket.io...');
+'use strict';
 
-socket.on('connect', function() {
-	viewer.setConnectionState(true);
-});
-
-socket.on('disconnect', function() {
-	socket.io.reconnect();
-	viewer.setConnectionState(false);
-});
+const Viewer = require('ftlabs-screens-viewer');
+const Carousel = require('ftlabs-screens-carousel');
+const port = location.port ? ':'+location.port : '';
+const host = '//'+location.hostname+port;
```

Comment:
I think `location.origin` would be a better fit.
`location.origin` returns the protocol, hostname and port --
"http://local.ft.com:3000"